fcgiwrap.c

```diff
@@ -578,6 +578,7 @@ static void handle_fcgi_request(void)
 			}
 
 			execl(filename, filename, (void *)NULL);
+			fprintf(stderr, "Error: %s\n", strerror(errno));
```

Comment:
What about being more verbose than just "Error"? "Failed to execute script" or something like:
```c
fprintf(stderr, "Failed to execute %s: %s\n", filename, strerror(errno));
```